### ISSUE TITLE
Fix long/many tags layout in challenge card 

### ISSUE THREAD

**Issue by rrchai** [Sage-Bionetworks/rocc-app#291]
Since the maximum topics we can take is 30 for now, find a way to better place many/long tags in current challenge card layout.

--> make them in a scrollable container and it will not break current card layout.

<img width="956" alt="Screen Shot 2021-09-24 at 1 43 43 PM" src="https://user-images.githubusercontent.com/73901500/134718502-968a1ac8-0b5e-48f3-970b-a546054d8006.png">

**Comment by tschaffter** [comment]
@rrchai  Notes:

- Reducing the maximum number of tags (30 was chosen arbitrarily). It's OK to define this limit depending on the UI design, e.g. depending on the space available to show them.
- scrollable content is probaly not a good idea here. Either the user won't notice that he/she can scroll OR there will be a scroll bar :/. All the information should be presented to the user without requiring him/her to perform unnecessarily actions.

**Comment by tschaffter** [comment]
We could also decrease the size of the font.

**Comment by rrchai** [comment]
@tschaffter  I agree not to avoid using scrollable content. I would suggest not to reduce the number of tags. The more tags, the more easily the challenges can be found. 

For pr #289, I made card only display the first four tags. We may display the most related/popular tags (with ontology system) in the future?
